### PR TITLE
Added PHP 8.2 support, dropped PHP 7 support 

### PR DESCRIPTION
**Pull request by glo71317**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | yes

### CODE REVIEW

**Comment by glo71317** [comment]
@Ocramius Can you please review this PR?

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": ">=7.4, <8.2"
+        "php": "~8.1.0 || ~8.2.0"
```
Comment (Ocramius) [inline composer.json]: IMO dependency range could stay wider, including at least PHP 8.0.

File: .laminas-ci.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.2": true
```
Comment (Ocramius) [inline .laminas-ci.json]: Could be dropped entirely, no?

**Review by Ocramius — CHANGES_REQUESTED** [review]
Question for the @laminas/technical-steering-committee: is it worth upgrading the bridge component for newer PHP versions?

We added explicit `conflict:` sections for components that were still using `Zend` namespace references, and older versions of those components are not compatible with PHP 8.2 anyway :thinking:

File: .laminas-ci.json (hunk 2 above)
Comment (glo71317) [reply .laminas-ci.json]: yes done

File: composer.json (hunk 1 above)
Comment (glo71317) [reply composer.json]: done

**Review by Ocramius — APPROVED** [review]
LGTM: worth shipping as-is 👍

**Comment by Ocramius** [comment]
Thanks @glo71317!

**Comment by boesing** [comment]
I'd say, that this will only work in case it is:
1. either installed as a standalone component in a project which does not require any laminas component
2. installed along with other components which are most probably conflicting with this component in a version which does not allow PHP 8.2

So option 1 would be a no-brainer: remove the dependency.
Option 2 would prevent developers to actually have a working `vendor` folder. Pretty sure there is no component which provides official PHP 8.2 support while still allowing the bridge.

**Comment by internalsystemerror** [comment]
Agreed here. I'm surprised this component isn't archived already, not sure where it could still be used on 8.2, but apparently someone is.

**Comment by Ocramius** [comment]
Hadn't considered standalone projects 🤔

**Pull request merged**